__tests__/jesttest.tsx

```diff
@@ -92,6 +94,55 @@ describe("LibraryContainer", function () {
     expect(libraryItem.prop('data').childItems[0].childItems[0].text).to.equal("Child1");
     expect(libraryItem.prop('data').childItems[0].childItems[1].text).to.equal("Child2");
   });
+   describe("Test event on libraryitem", function () {
+        it("Must recognise mouse event and change state", function () {
+          // test data to create LibraryItem 
+          let loadedTypesJson: any;
+          let layoutSpecsJson: any;
+          let libController: LibraryEntryPoint.LibraryController;
+          
+          // parent item that will hold the child items
+          let data = new ItemData("");
+          data.text = "TestItem";
+          data.itemType = "none";
+         
+          // create child items and link with parent
+          for (var i = 0; i < 2; i++) {
+            let item = new ItemData("");
+            item.text = "Child" + i;
+            item.itemType = "category";
+            data.appendChild(item);
+          }
+
+          // crate "LibraryContainer"" to pass as an argument for creation of "LibraryItem"
+          libController = LibraryEntryPoint.CreateLibraryController();
+          libController.on("itemClicked", function (item) {
+            console.log(item);
+          })
+
+          libController.on("searchTextUpdated", function (searchText) {
+            console.log(searchText);
+            return null;
+          });
+          let libContainer = libController.createLibraryContainer(layoutSpecsJson, loadedTypesJson);
+
+          // Mount is "real" rendering that will actually render your component into a browser environment. 
+          // If you are testing full React components, 
+          // mount is used to do rendering  and test actions are simulated on mounted html
+  
+          let libraryItem = mount(<LibraryItem libraryContainer={libContainer} data={data} />);
```

Comment:
Hi @fanwgwg, none of the above things that create `libContainer` is necessary. I think `LibraryItem` (and other components) requires `libContainer` for its `raiseEvent` kind of callback, can you guide @monikaprabhu in creating a helper method in this file that create a new type of `DummyLibraryContainer` class that does nothing (only to expose a few methods like `raiseEvent`, etc.)? Thanks!